src/auro-button.js

```diff
@@ -142,6 +161,12 @@ export class AuroButton extends LitElement {
 
   firstUpdated() {
     this.notifyReady();
+
+    this.addEventListener('auroButton-toggledText', () => {
```

Comment:
This may be me not understanding how Lit works but if I wanted to toggle `hideText` would I want to fire this event with the button as the target or would I update the `hideText` attribute? Are we supporting both avenues?

Reply:
Writing an event for this is something that Jason suggested. I am not sure exactly if there is a benefit to having an event, but yes we are currently supporting both options.

Reply:
@jason-capsule42 can you provide some of your wisdom on this pattern?

Reply:
@jordanjones243 let's pair program on this. This is a different thing then what I was suggesting.